### PR TITLE
Chapter 06 + new images

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: Ch06/CH06.md [hunk 1]
```diff
@@ -128,14 +128,14 @@ instance of RavenDB.
 The first thing we want to do is run an instance of RavenDB. I suggest running each RavenDB instance in its own terminal
 window, since that makes it easy to see the behavior of individual nodes:
 
-	./Server/Raven.Server.exe
-		--ServerUrl=http://127.0.0.1:8080
-		--Logs.Path=Logs/A 
+	./Server/Raven.Server.exe`
```
Comment (ayende) [inline Ch06/CH06.md:131]: Add a space before the ` here

File: Ch06/CH06.md [hunk 2]
```diff
@@ -144,20 +144,20 @@ Let's go into the database and create a couple of documents: "users/1" and "user
 the name. We're creating this database and those documents simply so we can see how we can grow a RavenDB cluster. 
 
 Now, we need to bring up another RavenDB node. Typically, you'd do that on a separate machine, but to make the demo
-easier, we'll just run it on the same node. Open a new terminal session and run the following command:
+easier, we'll just run it on the same machine. Open a new terminal session and run the following command:
 
-	./Server/Raven.Server.exe
-		--ServerUrl=http://127.0.0.2:8080
-		--Logs.Path=Logs/B
+	./Server/Raven.Server.exe`
```
Comment (ayende) [inline Ch06/CH06.md:149]: Add space before `

File: Ch06/CH06.md [hunk 3]
```diff
@@ -234,10 +233,10 @@ each database is independent. Each database instance will cooperate with the oth
...
 > When a node fails, the clients will transparently move to another node that holds the database they're connected to.
-> One of the tasks of the cluster's leader is to maintain the database topology, which clients will fetch as part of their 
-> initialization (and then keep current). We'll talk about failover from the client side more later in this chapter.
+> One of the tasks of the cluster's leader is to maintain the database topology, which clients fetch as part of their 
+> initialization (and then keep current). We'll talk more about failover from the client side later in this chapter.
 
-Now, you can start the first node by using the following command:
+Now, you can re-start the first node by using the following command:
 
 	./Server/Raven.Server.exe
```
Comment (ayende) [inline Ch06/CH06.md:241]: Should add ` here, no?

File: Ch06/CH06.md [hunk 4]
```diff
@@ -266,61 +265,61 @@ tasks later in this chapter.] and you'll not be able to deploy new indexes or mo
...
 
-You can probably guess what we're going to do now. In exactly the same way that we previously added node `B`, we're going to add
-the three new nodes. To do this, you'll execute the following commands:
+You can probably guess what we're going to do now. In exactly the same way that we previously added node `B`, we're going to add 
+three new nodes. To do this, execute the following commands:
 
-	./Server/Raven.Server.exe
-		--ServerUrl=http://127.0.0.3:8080
-		--Logs.Path=Logs/C 
+	./Server/Raven.Server.exe`
```
Comment (ayende) [inline Ch06/CH06.md:278]: Add space before `

File: Ch06/CH06.md [hunk 5]
```diff
@@ -266,61 +265,61 @@ tasks later in this chapter.] and you'll not be able to deploy new indexes or mo
...
+		--ServerUrl=http://127.0.0.3:8080`
+		--Logs.Path=Logs/C` 
 		--DataDir=Data/C
 
-	./Server/Raven.Server.exe
-			--ServerUrl=http://127.0.0.4:8080
-			--Logs.Path=Logs/D 
+	./Server/Raven.Server.exe`
```
Comment (ayende) [inline Ch06/CH06.md:283]: Add space before `

File: Ch06/CH06.md [hunk 6]
```diff
@@ -266,61 +265,61 @@ tasks later in this chapter.] and you'll not be able to deploy new indexes or mo
...
+			--ServerUrl=http://127.0.0.4:8080`
+			--Logs.Path=Logs/D` 
 			--DataDir=Data/D
 
-	./Server/Raven.Server.exe
-			--ServerUrl=http://127.0.0.5:8080
-			--Logs.Path=Logs/E 
+	./Server/Raven.Server.exe`
```
Comment (ayende) [inline Ch06/CH06.md:288]: Add space before `

File: Ch06/CH06.md [hunk 7]
```diff
@@ -735,46 +734,52 @@ for us to tell which version of the document is newer than the other. This is a
...
-* Kill node `A`. (It's running on `127.0.0.1`.) 
-* Bring up node `B` by executing `./Server/Raven.Server.exe --ServerUrl=http://127.0.0.2:8080 --Logs.Path=Logs/B --DataDir=Data/B`.
-* Modify the `users/1` document on node `B` so its name property will be "Doe," and save the document. You can see how this will look in
-  Figure 6.16.
-* Bring up node `A` by executing `./Server/Raven.Server.exe --ServerUrl=http://127.0.0.1:8080 --Logs.Path=Logs/A --DataDir=Data/A`.
+* Kill node `B`. (It's running on `127.0.0.2`) You can do that by simply closing the console window that's running it.  
+* Modify the `users/1` document on node `A` so its name property will be "John," and save the document. You can see how this will look in Figure 6.15.  
+* Kill node `A`. (It's running on `127.0.0.1`)  
+* Bring up node `B` by executing:  
+    ./Server/Raven.Server.exe`
```
Comment (ayende) [inline Ch06/CH06.md:757]: Add space before `

File: Ch06/CH06.md [hunk 8]
```diff
@@ -735,46 +734,52 @@ for us to tell which version of the document is newer than the other. This is a
...
+* Modify the `users/1` document on node `A` so its name property will be "John," and save the document. You can see how this will look in Figure 6.15.  
+* Kill node `A`. (It's running on `127.0.0.1`)  
+* Bring up node `B` by executing:  
+    ./Server/Raven.Server.exe`  
+    --ServerUrl=http://127.0.0.2:8080`  
+    --Logs.Path=Logs/B`  
+    --DataDir=Data/B  
+* Modify the `users/1` document on node `B` so its name property will be "Doe", and save the document. You can see how this will look in Figure 6.16.  
+* Bring up node `A` by executing:  
+    ./Server/Raven.Server.exe`
```
Comment (ayende) [inline Ch06/CH06.md:763]: Add space before `

Comment (ayende) [reply Ch06/CH06.md:763]: Also need to check here how this looks like after building the book

File: Ch06/CH06.md (hunk 1 above)
Comment (Danielle9897) [reply Ch06/CH06.md:131]: done

File: Ch06/CH06.md (hunk 2 above)
Comment (Danielle9897) [reply Ch06/CH06.md:149]: done

File: Ch06/CH06.md (hunk 3 above)
Comment (Danielle9897) [reply Ch06/CH06.md:241]: done

File: Ch06/CH06.md (hunk 4 above)
Comment (Danielle9897) [reply Ch06/CH06.md:278]: done

File: Ch06/CH06.md (hunk 5 above)
Comment (Danielle9897) [reply Ch06/CH06.md:283]: done

File: Ch06/CH06.md (hunk 6 above)
Comment (Danielle9897) [reply Ch06/CH06.md:288]: done

File: Ch06/CH06.md (hunk 8 above)
Comment (Danielle9897) [reply Ch06/CH06.md:763]: done

File: Ch06/CH06.md (hunk 7 above)
Comment (Danielle9897) [reply Ch06/CH06.md:757]: done

**Pull request merged**